vividus-plugin-azure-service-bus/src/main/java/org/vividus/azure/servicebus/service/ServiceBusService.java

```diff
@@ -0,0 +1,45 @@
+package org.vividus.azure.servicebus.service;
+
+import com.azure.core.amqp.AmqpRetryOptions;
+import com.azure.core.credential.TokenCredential;
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+import org.vividus.azure.servicebus.ServiceBusSendMessageException;
+import org.vividus.azure.servicebus.model.ServiceBusType;
+
+import java.time.Duration;
+
+public class ServiceBusService
+{
+    private ServiceBusClientBuilder.ServiceBusSenderClientBuilder clientBuilder;
+
+    public ServiceBusService(TokenCredential tokenCredential, String namespace, int maxRetries, Duration retryTimeout)
+    {
+        this.clientBuilder = new ServiceBusClientBuilder()
+                .fullyQualifiedNamespace(namespace + ".servicebus.windows.net")
+                .credential(tokenCredential)
+                .retryOptions(new AmqpRetryOptions().setMaxRetries(maxRetries).setTryTimeout(retryTimeout))
+                .sender();
+    }
+
+    public void send(ServiceBusType type, String name, String message)
+    {
+        if (type == ServiceBusType.QUEUE)
+        {
+            clientBuilder.queueName(name);
+        } else
+        {
+            clientBuilder.topicName(name);
+        }
+        try(ServiceBusSenderClient client = clientBuilder.buildClient())
```

Comment:
is client build re-usable? (in other words can you create multiple clients using a single client builder?